OracleDatabase/dockerfiles/12.1.0.2/installPerl.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Install latest Perl
+cd $INSTALL_DIR
+mv $ORACLE_HOME/perl $ORACLE_HOME/perl.old
+wget http://www.cpan.org/src/5.0/perl-5.14.1.tar.gz
```

Comment:
Well, I'd imagine (hope) the system Perl has already been updated to resolve this issue, but the Oracle Installer doesn't use it. Is there a way to get the OUI to use the system Perl instead of the embedded version?